### PR TITLE
Add sync arbiter chapter to book

### PR DESCRIPTION
**Pull request by Firstyear**

### CODE REVIEW

File: actix/src/sec-6-sync-arbiter.md [hunk 1]
```diff
@@ -1 +1,66 @@
-[WIP] 
\ No newline at end of file
+# SyncArbiter
+
+When you normally run Actors, there are multiple Actors sharing a single
```
Comment (DoumanAsh) [inline actix/src/sec-6-sync-arbiter.md]: >there are multiple Actors sharing a single

they are run on the same thread as System's owned event loop

File: actix/src/sec-6-sync-arbiter.md [hunk 2]
```diff
@@ -1 +1,66 @@
...
+thread and event loop in the System. However for CPU bound workloads, or
+highly concurrent workloads, you may wish to have a single Actor able
```
Comment (DoumanAsh) [inline actix/src/sec-6-sync-arbiter.md]: >you may wish to have a single Actor able

You may wish to offload Actor's work onto another threads

File: actix/src/sec-6-sync-arbiter.md [hunk 3]
```diff
@@ -1 +1,66 @@
...
+to run on a pool of OS threads.
+
+This is what a SyncArbiter provides - an enviroment allowing a single Actor
```
Comment (DoumanAsh) [inline actix/src/sec-6-sync-arbiter.md]: >enviroment 

I don't think it is right term. I suggest to re-phrase it without this word.

File: actix/src/sec-6-sync-arbiter.md [hunk 4]
```diff
@@ -1 +1,66 @@
...
+to be run on multiple threads concurrently.
+
+It's important to note a SyncArbiter can only host a single type of Actor. If
+you want to have two types of Actors running, you'll need to define two SyncArbiters
```
Comment (DoumanAsh) [inline actix/src/sec-6-sync-arbiter.md]: >you'll need to define two SyncArbiters

You don't define it, rather, you create SyncArbiter with corresponding actor.

So re-phrase it like:

You'll need to create each actor onto own SyncArbiter

File: actix/src/sec-6-sync-arbiter.md [hunk 5]
```diff
@@ -1 +1,66 @@
...
+
+## Creating a Sync Actor
+
+When constructing your Actor, you have to change `Context` to `SyncContext`
```
Comment (DoumanAsh) [inline actix/src/sec-6-sync-arbiter.md]: `When constructing your Actor,`

When implementing Actor?

`you have to change `Context` to `SyncContext`

It doesn't define that actor will run on `SyncArbiter`, instead it is `SyncArbiter` who requires that your Actor's Context type is `SyncContext`

File: actix/src/sec-6-sync-arbiter.md (hunk 2 above)
Comment (Firstyear) [reply actix/src/sec-6-sync-arbiter.md]: Sorry, I don't think this comment adds clarity, I find the wording confusing. I have however tweaked the section you have mentioned here so you may find it better now.

File: actix/src/sec-6-sync-arbiter.md (hunk 5 above)
Comment (Firstyear) [reply actix/src/sec-6-sync-arbiter.md]: Changed

**Review by DoumanAsh — APPROVED** [review]

**Pull request merged**